Makefile

```diff
@@ -32,7 +31,7 @@ COMMIT_HASH := $(shell git rev-parse HEAD)
 
 BTCD_COMMIT := $(shell cat go.mod | \
 		grep $(BTCD_PKG) | \
-		tail -n1 | \
+		head -n1 | \
```

Comment:
hm, was this wrong?

Reply:
It caused an error if you had a `replace` directive for `btcd`. The way we now compile the binary actually allows replace directives to be used. Only the unit tests still won't work with the replace (will pick what's in the upper part of `go.mod`). Maybe we should try to switch the unit tests over to an explicit binary as well and get rid of the `DEPGET` invocation for `btcd`?

Reply:
Sure, but could probably be done in another PR 👍

Reply:
Yeah, wasn't thinking about adding it to this PR.

Reply:
👍 ran into this the other day!